ssl/statem/extensions_srvr.c

```diff
@@ -89,7 +89,7 @@ int tls_parse_ctos_server_name(SSL *s, PACKET *pkt, unsigned int context,
     /*
      * Although the server_name extension was intended to be
      * extensible to new name types, RFC 4366 defined the
-     * syntax inextensibly and OpenSSL 1.0.x parses it as
+     * syntax extensibility and OpenSSL 1.0.x parses it as
```

Comment:
I don't know :-) I normally don't engage in discussions about natural language. This time I simply tried to act upon ping, but it seems to be touchy topic :-) Anyway, I for one see no place for noun, so it's definitely not last one. But again, it's only my head which is *not* exactly representative... First variant sounds right... It also might happen that any variant would be misinterpretation of RFC in question... In other words it might happen that it's not sufficient to concentrate on grammar...